db/seeds.rb

```diff
@@ -8,6 +8,7 @@
 
 Video.destroy_all
 Category.destroy_all
+User.destroy_all
```

Comment:
You do not need these `destroy_all` at all. You can rather drop and recreate database and seed with `rake db:setup`.